src/features/collapsible-sidebar/CollapsibleSidebarLogo.js

```diff
@@ -6,7 +6,7 @@ import styled from 'styled-components';
 import { injectIntl } from 'react-intl';
 import type { InjectIntlProvidedProps } from 'react-intl';
 import { bdlGridUnit, bdlBorderRadiusSizeLarge } from '../../styles/variables';
-import Logo from '../../icon/logo/BoxLogo';
```

Comment:
What's the reason for this icon switch?

My guess is we're trying to move our icons from:
`src/icons`
to
`src/icon`
as these come from the internal Box/design-assets repo.